docs/vmagent.md

```diff
@@ -231,6 +231,12 @@ This information may be useful for debugging target relabeling.
   by passing `-promscrape.suppressScrapeErrors` command-line flag to `vmagent`. The most recent scrape error per each target can be observed at `http://vmagent-host:8429/targets`
   and `http://vmagent-host:8429/api/v1/targets`.
 
+* For debugging scrapes targets relabeling process, you can observe status of targets at `/api/v1/targets` page.
+  You may need to increase value for the flag `--promscrape.maxDroppedTargets`.  
```

Comment:
can we introduce query paramter for that? changing the flag when you need more details looks like overkill

Reply:
query param doesn't work, since deleted scrape targets must be always tracked, so they always require additional memory for the tracking.